### PR TITLE
Floodscan + CCCM

### PR DESCRIPTION
**Pull request by turnerm**
Short analysis to compare floodscan data to CCCM

### CODE REVIEW

File: analysis/floodscan/01_floodscan.md [hunk 1]
```diff
@@ -1,25 +1,202 @@
...
 ```python
 %load_ext jupyter_black
 ```
 
 ```python
-from src import utils
+from datetime import timedelta
+
+import pandas as pd
+import matplotlib.pyplot as plt
```
Comment (zackarno) [inline analysis/floodscan/01_floodscan.md:14]: `matplotlib` should be added into `requirements.txt`?

File: analysis/floodscan/01_floodscan.md [hunk 2]
```diff
@@ -1,25 +1,202 @@
...
+    row.lon - pixel_window_size / 2, row.lon + pixel_window_size / 2
+)
+lat_slice = slice(
+    row.lat + pixel_window_size / 2, row.lat - pixel_window_size / 2
+)
+fs_raw.sel(lat=lat_slice, lon=lon_slice).mean(dim=["lat", "lon"]).max()
 ```
 
 ```python
-fs.time.max()
```
Comment (zackarno) [inline analysis/floodscan/01_floodscan.md:20]: probably not necessary, but i'd personally benefit from a description of this cells intent along with some comments

Comment (zackarno) [reply analysis/floodscan/01_floodscan.md:20]: just looked here this morning and realized I'm meant to be commenting on chunk starting on link 145, but from this comment it looks like line 20 - not sure why?

**Comment by zackarno** [comment]
nice work! looks great to me. After adding `ocha-anticipy` (no version) to requirements and `import np` in the `floodscan.md` doc the code worked well and is clear to me.  With those small changes I should be good to merge!

File: analysis/floodscan/01_floodscan.md (hunk 1 above)
Comment (turnerm) [reply analysis/floodscan/01_floodscan.md:14]: good catch, fixed

File: analysis/floodscan/01_floodscan.md [hunk 3]
```diff
@@ -1,25 +1,202 @@
...
+    df_rp["rank"] = np.arange(n) + 1
+    df_rp["exceedance_probability"] = df_rp["rank"] / (n + 1)
+    df_rp["rp"] = 1 / df_rp["exceedance_probability"]
+    return interp1d(df_rp["rp"], df_rp[rp_var])
 ```
 
 ```python
-fs.plot("time", "sum")
+lon_slice = slice(
+    row.lon - pixel_window_size / 2, row.lon + pixel_window_size / 2
```
[the remark this answers is not in the document]
Comment (turnerm) [reply analysis/floodscan/01_floodscan.md:136]: oops so sorry this cell was meant to be deleted!

File: analysis/floodscan/01_floodscan.md (hunk 2 above)
Comment (turnerm) [reply analysis/floodscan/01_floodscan.md]: Sorry Zack you're so right, the end of the notebook was pretty incomprehensible. I added lots of comments!

**Pull request merged**